### ISSUE TITLE
BINARY fields throw decode error

### ISSUE THREAD

**Issue by kived** [dbcli/mycli#122]
We have many BINARY(16) columns in our database. When trying to select from these tables, mycli encounters a decode error:
````
'ascii' codec can't decode byte 0xc0 in position 2: ordinal not in range(128)
````

The best approach would be to automatically `HEX()` any BINARY fields for display (like HeidiSQL does). And that would be just one more thing that mycli would have over the official mysql client.

**Comment by amjith** [comment]
Yes. There is already work underway to do exactly this. #106 

Thanks to @danieljwest.

**Comment by danieljwest** [comment]
Good to know hex was the right way to go!

**Comment by mdsrosa** [comment]
I'm closing this issue since it has already been resolved by #132. :wink:
